src/App.jsx

```diff
@@ -10,74 +15,67 @@ export const goods = [
   'Bread',
   'Fish',
   'Honey',
-  'Jam',
+  DEFAULT_GOOD,
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
-
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export const App = () => {
+  const [selectedGood, setGood] = useState(DEFAULT_GOOD);
+  const isSelectedGood = good => good === selectedGood;
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood !== NO_GOOD_SELECTED
+          ? (
+            <>
+              {`${selectedGood} is selected`}
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+              <button
+                data-cy="ClearButton"
+                type="button"
+                className="delete ml-3"
+                onClick={() => setGood(NO_GOOD_SELECTED)}
+              />
+            </>
+          )
+          : (
+            'No goods selected'
+          )}
+      </h1>
 
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
+      <table className="table">
+        <tbody>
+          {goods.map(good => (
+            <tr
+              data-cy="Good"
+              key={good}
+              className={cn({
+                'has-background-success-light': isSelectedGood(good),
+              })}
             >
-              +
-            </button>
-          </td>
+              <td>
+                <button
+                  data-cy={isSelectedGood(good) ? 'RemoveButton' : 'AddButton'}
```

Comment:
Instead of calling function we can create a constant value that will have info if good is selected